server/game/index.js

```diff
@@ -34,6 +34,44 @@ exports.addNewUser = async (userId, userSocketId, room) => {
   }
 };
 
+const getRandomIndices = (word) => {
+  // Random indexes to show hint letters
+  const randomIndex = (arr, length) => {
+    const randIndex = Math.floor(Math.random() * length);
+    if (!arr.includes(randIndex)) {
+      return randIndex;
+    }
+    return randomIndex(arr, length);
+  };
+
+  const guessWordLength = word.length;
+  let randomIndices = [];
+
+  // Show one-third of the guess word
+  for (let i = 0; i < Math.ceil(guessWordLength / 3); i++) {
+    randomIndices = [
+      ...randomIndices,
+      randomIndex(randomIndices, guessWordLength)
+    ];
+  }
+  return randomIndices;
+}
+
+exports.getMaskedHintWord = (word) => {
```

Comment:
@sampathBlam what do you think?
Is this required to be exported? Anyway, I think we have to start moving the auxiliary features outside of game/index.js so that we dont end up a gigantic game loop file.

Reply:
Yes can we move it to a separate file ?